server/src/db/seeders/MemeSeeder.js

```diff
@@ -0,0 +1,31 @@
+import { Meme  } from "../../models/index.js"
+
+class MemeSeeder {
+    static async seed() {
+        await Meme.query().delete();
+
+        const firstMeme = await Meme.query().insert({
+        title: "First Meme",
+        memeUrl: "https://i.kym-cdn.com/entries/icons/original/000/018/012/this_is_fine.jpeg",
+        userId: firstUser.id,
```

Comment:
each of these properties should be indented in (line 8 - 10). Same should be done for the other objects below

Reply:
Done, this was a copy and paste oversight